tests/tests_integration/test_api/test_workflows.py

```diff
@@ -88,22 +88,27 @@ def new_workflow(cognite_client: CogniteClient, data_set: DataSet):
 
 
 @pytest.fixture(scope="class")
-def workflow_list(cognite_client: CogniteClient, data_set: DataSet):
+def workflow_list(cognite_client: CogniteClient, data_set: DataSet) -> WorkflowList:
```

Comment:
To make it clear that this is not created on the fly.
```suggestion
def persisted_workflow_list(cognite_client: CogniteClient, data_set: DataSet) -> WorkflowList:
```